modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -2060,7 +2063,7 @@ private void stopCacheSafely(GridCacheContext<?, ?> cctx, boolean clearDbObjects
             prepareCacheStop(cctx.name(), false, clearDbObjects);
 
             if (!cctx.group().hasCaches())
-                stopCacheGroup(cctx.group().groupId());
+                stopCacheGroup(cctx.group().groupId(), clearDbObjects);
```

Comment:
Here `clearDbObjects` is incorrect because `destroy == false`. IMHO valid value for flag is `false`.

Reply:
Fixed.